gherkin-utils/java/gherkin-utils/pom.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>gherkin</groupId>
+  <artifactId>gherkin-utils</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>gherkin-utils</name>
+  <url>https://github.com/cucumber/cucumber/tree/master/gherkin-utils</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
+      <scope>test</scope>
+    </dependency>
+      <dependency>
+          <groupId>io.cucumber</groupId>
+          <artifactId>messages</artifactId>
+          <version>13.1.0</version>
+      </dependency>
+      <dependency>
+          <groupId>org.junit.jupiter</groupId>
+          <artifactId>junit-jupiter</artifactId>
+          <version>RELEASE</version>
```

Comment:
The `RELEASE` and `LATEST` tags have been deprecated by Maven. They may also lead to unstable builds. Please use a version number instead.

Reply:
I have changed it to the last version 5.7.0